examples/with-next/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     ],
     "paths": {
       "@/*": ["./*"]
-    }
+    },
+    "target": "ES2017"
```

Comment:
Does this enable any particular feature?

Reply:
I need to remove all the noise, I just added it by reflex